ext/ManifoldsDistributionsExt/distributions.jl

```diff
@@ -0,0 +1,128 @@
+"""
+    FVectorvariate
+
+Structure that subtypes `VariateForm`, indicating that a single sample
+is a vector from a fiber of a vector bundle.
+"""
+struct FVectorvariate <: VariateForm end
```

Comment:
Is that a good idea to introduce new types in extensions? I would prefer to only add new methods to functions in extensions.

Reply:
It's not a great design but it's an intermediate thing until we have "manifold distributions done right" in a separate package. Putting these types there makes them private. If anyone relies on that they can still use it but I would discourage any new uses.

Reply:
Hm, ok, I get your point. I am not a fan of this, but I do see, that we would need someone to do distributions right. Let‘s find someone ;)

Reply:
I fully support your idea of finding someone to do distributions right :wink:

Reply:
We could try to ask in the forum. Besides that we could try to convince one of my students next year to work a bit on that, but that is probably more like for us to get into that then.

Reply:
I think it's not a good student project because it's in part an API development problem. There is also lots of confusing papers with misleading claims on this topic. Anyway, I'm not going to be looking for anyone myself but if you find someone, I could guide them.

Reply:
That‘s why I said they probably can bring me only to know a bit more about it, since it will be quite some API design. 

Just out of interest I might ask in the forum if someone would be interested to join. but its also not my main focus, I am ok with our current `rand(M)` stuff, that is what I mainly need.

Reply:
I just felt like opening something here https://discourse.julialang.org/t/call-for-contributors-distributions-on-manifolds/117820 – let‘s see, whether someone would be interested in something like a ManifoldDistributions.